gui/lib/widgets/popups/decision_popup.dart

```diff
@@ -33,20 +34,43 @@ final class DecisionPopup extends Popup {
   }
 
   Widget _noButton(BuildContext context) {
-    return OutlinedButton(
-      onPressed: () => closePopup(context),
-      child: Text(decisionMetadata.noButtonText),
+    return SizedBox(
+      height: 32,
+      child: OutlinedButton(
+        style: OutlinedButton.styleFrom(
+          padding: const EdgeInsets.symmetric(horizontal: 12, vertical: 6),
+        ),
+        onPressed: () => closePopup(context),
+        child: Text(decisionMetadata.noButtonText),
+      ),
     );
   }
 
   Widget _yesButton(WidgetRef ref, BuildContext context) {
-    return ElevatedButton(
-      child: Text(decisionMetadata.yesButtonText),
-      onPressed: () async {
-        await decisionMetadata.yesAction(ref);
-        if (!context.mounted) return;
-        closePopup(context);
-      },
+    return SizedBox(
+      height: 32,
+      child: ElevatedButton(
+        onPressed: () async {
+          if (decisionMetadata.autoClose) {
```

Comment:
[small] maybe it's enough to have `navigateToRoute` only without having `autoClose` since `navigateToRoute` requires `autoClose`, so now if you want to navigate to other screen from popup, you also have to define `autoClose: true` along with `navigateToRoute`